commit f1ac97e11ce50e9eaf4308580abe9cbef4004f5e (refs/changes/15/715/1)
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2016-10-17T12:31:02-07:00 (3 years ago)
    
    DOCKER-950 docker pull fails on registry that does not use authentication
    Reviewed by: Trent Mick <trentm@gmail.com>

diff --git a/CHANGES.md b/CHANGES.md
index 0b4c881..6c8e57f 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # IMGAPI changelog
 
+## 3.1.2
+
+- DOCKER-950 docker pull fails on registry that does not use authentication
+
 ## 3.1.1
 
 - DOCKER-663 Support Amazon ECR Registry
diff --git a/package.json b/package.json
index c3d91f6..f0d65f6 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "imgapi",
   "description": "Image API to manage images for SDC 7",
-  "version": "3.1.1",
+  "version": "3.1.2",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
@@ -10,7 +10,7 @@
     "bunyan": "1.8.1",
     "cmdln": "3.2.1",
     "dashdash": "1.10.0",
-    "docker-registry-client": "3.2.1",
+    "docker-registry-client": "3.2.2",
     "effluent-logger": "git+https://github.com/joshwilsdon/effluent-logger.git#d662f161a07f94045ad2afb45442931511c40e51",
     "expiring-lru-cache": "2.1.0",
     "extsprintf": "1.2.0",
